recipes/debugging/vs-code-debug.md

```diff
@@ -1,40 +1,57 @@
-# VS Code Amber Debug
+# Debugging Crystal Projects using VSCode
 
-Debugging an Amber project on VSCode with GDB and [Native Debug](https://github.com/crystal-lang-tools/vscode-crystal-lang/wiki/Useful-extensions#debugging).
+This tutorial have some tips and tricks to debug Crystal projects with tools like GDB or LLDB using debugger clients like [Native Debug](https://marketplace.visualstudio.com/items?itemName=webfreak.debug) for VSCode.
```

Comment:
This tutorial has tips and tricks on how to debug Crystal projects. It shows how to leverage tools like GDB or LLDB using debugger clients like [Native Debug](https://marketplace.visualstudio.com/items?itemName=webfreak.debug) for VSCode.